### PR TITLE
MULE-10585: Make the container ThreadPools configurable

### PR DESCRIPTION
**Pull request by elrodro83**

### CODE REVIEW

File: distributions/standalone/src/main/resources/conf/scheduler-pools.conf [hunk 1]
```diff
@@ -11,17 +13,26 @@ org.mule.runtime.scheduler.gracefulShutdownTimeout=15000
 # The number of threads to keep in the cpu_lite pool, even if they are idle.
 # Supports Expressions
 org.mule.runtime.scheduler.cpuLight.threadPool.size=2*cores
+# The size of the queue to use for holding cpu_lite tasks before they are executed.
+# Supports Expressions
+org.mule.runtime.scheduler.cpuLight.workQueue.size=mem / (2*3*32)
```
Comment (dfeist) [inline distributions/standalone/src/main/resources/conf/scheduler-pools.conf]: Why not simply `cpuLight.queueSize`, are there, or will there be, other properties of the queue?

File: distributions/standalone/src/main/resources/conf/scheduler-pools.conf [hunk 2]
```diff
@@ -11,17 +13,26 @@ org.mule.runtime.scheduler.gracefulShutdownTimeout=15000
...
 
 # The number of threads to keep in the I/O pool.
 # Supports Expressions
 org.mule.runtime.scheduler.io.threadPool.coreSize=cores
 # The maximum number of threads to allow in the I/O pool.
 # Supports Expressions
-org.mule.runtime.scheduler.io.threadPool.maxSize=cores*cores
+org.mule.runtime.scheduler.io.threadPool.maxSize=mem / (2*30*64)
```
Comment (dfeist) [inline distributions/standalone/src/main/resources/conf/scheduler-pools.conf]: Where do these values come from?  Why should size be based on memory available? IMO we should just be using fixed values, or potentially a multiple `threadPool.maxSize` of threads for these, at least until testing.

File: distributions/standalone/src/main/resources/conf/scheduler-pools.conf [hunk 3]
```diff
@@ -11,17 +13,26 @@ org.mule.runtime.scheduler.gracefulShutdownTimeout=15000
...
+# The size of the queue to use for holding I/O tasks before they are executed.
+# Supports Expressions
+org.mule.runtime.scheduler.io.workQueue.size=mem / (2*3*32)
```
Comment (dfeist) [inline distributions/standalone/src/main/resources/conf/scheduler-pools.conf]: Where do these values come from? Why should size be based on memory available? IMO we should just be using fixed values, or potentially a multiple threadPool.maxSize for these, at least until testing.

File: distributions/standalone/src/main/resources/conf/scheduler-pools.conf (hunk 1 above)
Comment (elrodro83) [inline distributions/standalone/src/main/resources/conf/scheduler-pools.conf]: could be, but at least this is consistent with the pool size property

**Pull request merged**